README.md

```diff
@@ -1,5 +1,24 @@
-# Toha Guides
+# Les guides du thème Toha en langue française
```

Comment:
We can't translate the README file. There can be only one README file. That should be in English. Can you please, make the changes in English?

Reply:
Hi @hossainemruz,

My README file is not a translation. I had reset it for my own repo. What I'm going to do, I'm going to put it back to original. No problem.